src/api/admin/index.js

```diff
@@ -1,9 +1,24 @@
 const { Router } = require('express');
-const { getAdmin, postAdmin } = require('./admin.ctrl');
+const { isAuthenticated } = require('../auth/passport');
+const {
+    getUser,
+    postUser,
+    postUserPermission,
+    deleteUser,
+    postNotice,
+    postEditNotice,
+    deleteNotice,
+} = require('./admin.ctrl');
 
 const router = Router();
 
-router.get('/:user_id', getAdmin);
-router.post('/', postAdmin);
+router.get('/user', isAuthenticated, getUser); // 사용자 리스트
+router.post('/user', isAuthenticated, postUser); // 사용자 추가
+router.post('/user/permission', isAuthenticated, postUserPermission); // 사용자 등급 변경
```

Comment:
이건
```suggestion
router.put('/user/permission', isAuthenticated, postUserPermission); // 사용자 등급 변경
```
 이 더 맞는 방법일 거 같음